common/test/acceptance/tests/studio/test_studio_split_test.py

```diff
@@ -333,7 +333,7 @@ def create_group_configuration_experiment(self, groups, associate_experiment):
             return config, split_test
         return config
 
-    def publish_unit_in_LMS_and_view(self, courseware_page):
+    def publish_unit_in_lms_and_view(self, courseware_page, publish=True):
         """
         Given course outline page, publish first unit and view it in LMS
```

Comment:
You should update the doc-string to indicate that when publish is false, it will only view.